elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/queryengines/sql/SQLEntityHydrator.java

```diff
@@ -9,78 +9,37 @@
-import com.yahoo.elide.utils.coerce.CoerceUtil;
-import lombok.AccessLevel;
-import lombok.Getter;
 
-import java.util.AbstractMap;
+import java.sql.ResultSet;
 import java.util.Collections;
-import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
-import java.util.stream.Collectors;
-
-import javax.persistence.EntityManager;
 
 /**
  * {@link SQLEntityHydrator} hydrates the entity loaded by
  * {@link QueryEngine#executeQuery(Query, QueryEngine.Transaction)}.
  */
 public class SQLEntityHydrator extends AbstractEntityHydrator {
 
-    @Getter(AccessLevel.PRIVATE)
-    private final EntityManager entityManager;
-
     /**
      * Constructor.
      *
      * @param results The loaded objects from {@link QueryEngine#executeQuery(Query, QueryEngine.Transaction)}
      * @param query  The query passed to {@link QueryEngine#executeQuery(Query, QueryEngine.Transaction)} to load the
      *               objects
      * @param entityDictionary  An object that sets entity instance values and provides entity metadata info
-     * @param entityManager  An service that issues JPQL queries to load relationship objects
      */
     public SQLEntityHydrator(
-            List<Object> results,
+            ResultSet results,
             Query query,
-            EntityDictionary entityDictionary,
-            EntityManager entityManager
+            EntityDictionary entityDictionary
     ) {
         super(results, query, entityDictionary);
-        this.entityManager = entityManager;
     }
 
     @Override
-    protected Map<Object, Object> getRelationshipValues(
-            Class<?> relationshipType,
-            List<Object> joinFieldIds
-    ) {
-        if (joinFieldIds.isEmpty()) {
-            return Collections.emptyMap();
-        }
-
-        List<Object> uniqueIds = joinFieldIds.stream()
-                .distinct()
-                .collect(Collectors.toCollection(LinkedList::new));
-
-        List<Object> loaded = getEntityManager()
-                .createQuery(
-                        String.format(
-                                "SELECT e FROM %s e WHERE %s IN (:idList)",
-                                relationshipType.getCanonicalName(),
-                                getEntityDictionary().getIdFieldName(relationshipType)
-                        )
-                )
-                .setParameter("idList", uniqueIds)
-                .getResultList();
-
-        return loaded.stream()
-                .map(obj -> new AbstractMap.SimpleImmutableEntry<>(
-                        CoerceUtil.coerce(
-                                (Object) getEntityDictionary().getId(obj),
-                                getEntityDictionary().getIdType(relationshipType)
-                        ),
-                        obj))
-                .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
+    protected Map<Object, Object> getRelationshipValues(Class<?> relationshipType, List<Object> joinFieldIds) {
```

Comment:
since there are no plans to support this, we can remove it from here and the abstract class.

Reply:
removed related methods.